code/stack/model/stack.py

```diff
@@ -0,0 +1,42 @@
+class Stack:
+    max_size = 100
```

Comment:
It looks like `max_size` and `self. max_size` are different variables. I think `self. max_size`  in constructor is enough.

Reply:
Thanks, I have applied your tip.